website/src/components/SequenceDetailsPage/getTableData.ts

```diff
@@ -157,6 +157,9 @@ function toTableData(config: Schema) {
                 value: mapValueToDisplayedValue(details[metadata.name], metadata),
                 header: metadata.header ?? '',
                 type: { kind: 'metadata', metadataType: metadata.type },
+                ...(metadata.orderOnDetailsPage !== undefined
+                    ? { orderOnDetailsPage: metadata.orderOnDetailsPage }
+                    : {}),
```

Comment:
That seems a bit unpleasant to read, can we simplify it and write `orderOnDetailsPage: metadata.orderOnDetailsPage`?

Reply:
Yeah there is a difference between no-key-set and key-set-but-value-is-undefined however your change satisfies the type-checker so seems all good!